src/http.ts

```diff
@@ -0,0 +1,28 @@
+import type { AnySchema } from "joi"
+import fetch from "node-fetch"
+
+export const validatedFetch = async <T>(
```

Comment:
how is this used?
what does it return?

Reply:
> how is this used?

https://github.com/paritytech/command-bot/search?q=validatedFetch

> what does it return?

It returns the T

We don't specify return types in general. There's a lint to be enabled for that: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md, but it's turned off right now. Do you wish to turn it on?